client/src/components/SaladListComponent.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from "react"
+import SaladTile from "./SaladTile"
+
+
+const SaladListComponent = (props) => {
+
+    const [salads, setSalads] = useState([])
+
+    const getSalads = async() =>{
```

Comment:
should have a space before opening braces - `=> {`
https://github.com/airbnb/javascript#whitespace--before-blocks

should add a space before parenthesis - `async ()`
https://github.com/airbnb/javascript#whitespace--around-keywords